drivers/hmis/spec/system/hmis/enrollment_management_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+require_relative '../../requests/hmis/login_and_permissions'
+require_relative '../../support/hmis_base_setup'
+
+RSpec.feature 'Enrollment/household management', type: :system do
+  include_context 'hmis base setup'
+  # could parse CAPYBARA_APP_HOST
+  let!(:ds1) { create(:hmis_data_source, hmis: 'localhost') }
+  let(:c1) { create :hmis_hud_client, data_source: ds1, user: u1, last_name: 'Aabcdefghij' }
+  let(:c2) { create :hmis_hud_client, data_source: ds1, user: u1, last_name: 'Klmnopqrs' }
+  let!(:unit) { create :hmis_unit, project: p1, user: user }
```

Comment:
> This doesn't make a ton of sense because the clients are in a different data source from the project, right?

Hmm, I expect the override of ds1 on line 8 applies across all contexts (I think it just redefines the accessor method under the hood). I don't think we could get two definitions for ds1 here but I'll double check that. This test passed when I ran it before but I could well be missing something

Reply:
huh okay. I should really understand better how rspec variables work. it does look like it's passing!

Reply:
yeah, looks like there's only one ds1. Could adjust for clarity if needed